src/ducks/categories/categoriesMap.js

```diff
@@ -6,45 +6,51 @@
 
 import tree from './tree'
 
+const getCssVariableValue = variableName =>
```

Comment:
Shouldn't it be placed in a more global place? I think we will reuse it elsewhere pretty quickly.

Reply:
https://github.com/cozy/cozy-ui/pull/763/files